docs/basic/getting-started/basic-type-examples.md

```diff
@@ -59,12 +59,8 @@ Relevant for components that accept other React components as props.
 
 ```tsx
 export declare interface AppProps {
-  children1: JSX.Element; // bad, doesnt account for arrays
-  children2: JSX.Element | JSX.Element[]; // meh, doesn't accept strings
-  children3: React.ReactChildren; // despite the name, not at all an appropriate type; it is a utility
-  children4: React.ReactChild[]; // better, accepts array children
-  children: React.ReactNode; // best, accepts everything (see edge case below)
-  functionChildren: (name: string) => React.ReactNode; // recommended function as a child render prop type
```

Comment:
There's no sensible recommendation for render prop type since it depends heavily on what you want to do. The pattern itself is rarely used nowadays so let's not overwhelm people with React patterns at this point.